cpp/src/Slice/Parser.h

```diff
@@ -1039,11 +1046,12 @@ namespace Slice
         [[nodiscard]] std::set<std::string> getTopLevelModules(const std::string& file) const;
 
     private:
-        Unit(bool all, MetadataList defaultFileMetadata);
+        Unit(std::string_view languagePrefix, bool all, MetadataList defaultFileMetadata);
```

Comment:
I find this use of string_view over the top. Using strings is preferable, especially for something you adopt.

Reply:
Fine, I switched it to use `string` instead.